### PR TITLE
Prepare packages for release based on Core 1.22.0 package

### PR DESCRIPTION
**Pull request by annelo-msft**
Moving library project references to package references now that [Azure.Core 1.22.0](https://www.nuget.org/packages/Azure.Core/1.22.0) has been released.

### CODE REVIEW

File: eng/Packages.Data.props [hunk 1]
```diff
@@ -81,7 +81,7 @@
 
     <!-- Azure SDK packages -->
     <PackageReference Update="Azure.Communication.Common" Version="1.0.1" />
-    <PackageReference Update="Azure.Core" Version="1.21.0" />
+    <PackageReference Update="Azure.Core" Version="1.22.0" />
     <PackageReference Update="Azure.Core.Amqp" Version="1.2.0" />
     <PackageReference Update="Azure.Core.Experimental" Version="0.1.0-preview.18" />
```
Comment (AlexGhiondea) [inline eng/Packages.Data.props:86]: This should be bumped as well to .19.

**Review by pakrym — APPROVED** [review]

**Review by christothes — APPROVED** [review]
✅ Tables
✅ Confidential Ledger

**Comment by Mohit-Chakraborty** [comment]
I noticed some project refs to `Azure.Core.Experimental` are kept as-is while some have been switched over to package refs.
Is there an upcoming release of `Azure.Core.Experimental` when all refs can be switched over?

**Review by Mohit-Chakraborty — APPROVED** [review]

**Review by jsquire — APPROVED** [review]

**Comment by annelo-msft** [comment]
> I noticed some project refs to `Azure.Core.Experimental` are kept as-is while some are being switched over to package refs.
> Is there an upcoming release of `Azure.Core.Experimental` when all refs can be switched over?

Nice catch, @Mohit-Chakraborty, thank you!

**Comment by JoshLove-msft** [comment]
It looks like you are also removing Azure.Core and Azure.Core.Experimental from all solutions. Is there a reason for this?

**Comment by Mohit-Chakraborty** [comment]
> It looks like you are also removing Azure.Core and Azure.Core.Experimental from all solutions. Is there a reason for this?

I am guessing it is to prevent folks from getting confused when the source in `Core` doesn't match the binaries from the package ref.

**Comment by JoshLove-msft** [comment]
> > It looks like you are also removing Azure.Core and Azure.Core.Experimental from all solutions. Is there a reason for this?
> 
> I am guessing it is to prevent folks from getting confused when the source in `Core` doesn't match the binaries from the package ref.

That is fair, but the project has always been in many of these solutions. Personally, I like that Azure.Core is in the solutions so I can quickly jump to the code to check something. I think this convenience outweighs any potential confusion about differing versions - usually the changes in each release are pretty targeted and only involve API additions. 

Happy to be outvoted if others prefer removing Azure.Core.

**Comment by jsquire** [comment]
> Happy to be outvoted if others prefer removing Azure.Core.

For what it's worth, my vote goes to "if it's not used, it shouldn't be in the solution."

**Comment by pakrym** [comment]
IMO partners don't have a good understanding of how project/package ref switching works. Some might try to debug azure.core but find that breakpoints are not hit or changes are not applied.

**Comment by Mohit-Chakraborty** [comment]
> Personally, I like that Azure.Core is in the solutions so I can quickly jump to the code to check something. I think this convenience outweighs any potential confusion about differing versions

I keep vacillating between the two styles of maintaining solutions. I also like the convenience of being able to browse the source of core libraries my project depends upon. Then, sometimes I worry I cannot trust any decisions I make from looking at the code because there is a chance, howsoever small, that the code has changed.

For long running ownership of projects, sometimes I maintain a private and local .sln file where I add all projects that I find interesting.

**Pull request merged**